src/chainstate/stacks/index/storage.rs

```diff
@@ -408,15 +408,16 @@ impl TrieFileStorage {
             }
         }
 
-        let partially_written_state = TrieFileStorage::scan_tmp_blocks(dir_path)?;
+        let dir_path = dir_path.to_string();
+        let partially_written_state = TrieFileStorage::scan_tmp_blocks(&dir_path)?;
```

Comment:
Might as well have `TrieFileStorage::scan_tmp_blocks()` take a `&str` instead of a `&String` if it will make things easier.

Reply:
It would make the interfaces more consistent, but `to_string` (or the prior clone) is still necessary in any event since there's a struct that wants an owned string later on.